### ISSUE TITLE
Feature Request: Feed scheduling

### ISSUE THREAD

**Issue by Popsiclestick** [CERT-BDF/Hippocampe#31]
### Feature Request

While reading through the documentation I came across [this section](https://github.com/CERT-BDF/Hippocampe/blob/master/docs/tutorial.md#retrieve-feeds-automatically-and-periodically)

This seems very brittle and transient. I think removing this as a hidden API call, and making this a configuration option for the feed in '../Hippocampe/core/conf/feeds' might be a more useful feature since thats where we configure the feeds we're going to download.

Keeping the API call + allowing it configurable via feed configurations also seems solid.

**Comment by ninSmith** [comment]
Hi Popsiclestick,

Thank you for your FR.

However, we are not sure to understand. 
Just to be sure, you would like to configure the indexation periodicity in the feed's conf file, right ?

**Comment by Popsiclestick** [comment]
Yes. I think creating an option in ```core/conf/feeds/<feed>.conf``` which allows you to specify a variable that lets you set how often a feed is redownloaded (Something like ```sync : 12h ```).

**Comment by ninSmith** [comment]
Hum..

We then would calculate the duration between feed's ```lastQuery``` field and time of indexation.
Compare the duration with ```sync``` value and depending on the result update or not.

We sure could implement that.

First we need to figure out whether we keep the ```sync``` value **AND** the API call or only one of these two...

Will keep you updated.

Thanks for your idea

**Comment by darksheer** [comment]
Will +1 this as some feeds are actually updated on an hourly basis, some 24 hours, some unknown.  Having more aggressive pulls for faster-updated feeds makes a lot of sense.
